docs/support/project-structure.md

```diff
@@ -89,6 +89,10 @@ are separated out into their own folder, see further down.
   There are no "core" packages in the backend. Instead we have `backend-common`
   which contains helper middleware and other utils.
 
+- [`catalog-client`](https://github.com/backstage/backstage/tree/master/packages/catalog-client) -
+  A client to communicate with Backstage software catalog. Recommended for backend plugins, compatible
```

Comment:
Maybe we can rephrase this to something like:

```
An isomorphic client to interact with the Software Catalog. For use with backend plugins directly, and frontends by using `@backstage/plugin-catalog` in combination with `useApi` and the `catalogApiRef`
```

Reply:
@benjdlambert Yeep, very well. Done!